Makefile

```diff
@@ -12,14 +14,21 @@ THIS_MAKEFILE_PATH := $(word $(words $(MAKEFILE_LIST)),$(MAKEFILE_LIST))
 THIS_DIR := $(shell cd $(dir $(THIS_MAKEFILE_PATH));pwd)
 NPM_BIN_DIR = $(shell npm bin)
 
+NPMRC_ELECTRON_VERSION := $(shell npm config get target)
+PACKAGE_ELECTRON_VERSION := $(shell node -e "console.log(require('./package.json').devDependencies.electron)")
+
 RED = `tput setaf 1`
 GREEN = `tput setaf 2`
 CYAN = `tput setaf 6`
 RESET = `tput sgr0`
 
 CALYPSO_DIR := $(THIS_DIR)/calypso
 
-CHECKMARK = ✓
+ifeq ($(OS),Windows_NT)
+	CHECKMARK = OK
+else
+	CHECKMARK = ✓
+endif
```

Comment:
if we only use `OK` we won't need this variable at all and then the output will likely be a little more consistent across platforms

Reply:
Amended in 2a57d82.